text_to_sql_plus_RAG_SanyogMishra/ollama_integration.py

```
@@ -0,0 +1,80 @@
+from llama_index.core import Settings
+from llama_index.embeddings.ollama import OllamaEmbedding
+from llama_index.llms.ollama import Ollama
+
+def is_ollama_running(ollama_base_url="http://localhost:11434"):
+    """
+    Check if Ollama server is running by testing the /api/tags endpoint.
+    """
+    try:
+        response = requests.get(f"{ollama_base_url}/api/tags")
+        return (response.status_code == 200)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)